src/test/ui/simd-intrinsic/simd-intrinsic-generic-shuffle.rs

```diff
@@ -0,0 +1,33 @@
+// build-fail
+
+// Test that the simd_shuffle intrinsic produces ok-ish error
+// messages when misused.
```

Comment:
Are these error messages going to be the same for whatever wraps these intrinsics? If so, the diagnostics should definitely be better than ok-ish, I feel.

Reply:
Honestly, I just copied this file from another in this directory 🙂 This error should be completely prevented by the type system in `std::simd`, so this error should never make it to a user.  Actually, it was decided a while back that `std::simd` should never produce monomorphisation errors.

Reply:
I guess its fine to not add a test like this at all, in that case, I feel, as its only an implementation detail, but I guess the cost of such a test is pretty low regardless.

Reply:
For now, it's useful to test the intrinsics because `std::simd` isn't in tree yet.  After it is, you are probably right.

Reply:
Yeah, I would rather test that we generate a distinct error at all if something slips through, but it should "never occur" for external users who use the intended API, so from our perspective anyone deliberately using this is opting into the Bad Error.